src/components/EditableHeading/EditableHeading.jsx

```diff
@@ -246,7 +261,10 @@ EditableHeading.defaultProps = {
   ignoreBlurClass: undefined,
   onFinishEditing: undefined,
   onCancelEditing: undefined,
-  onIgnoreBlurEvent: undefined
+  onIgnoreBlurEvent: undefined,
+  style: undefined,
```

Comment:
did you use this props? if not can you please remove it

Reply:
I'm not using them, but how can we know that others have not used them already.
or do you mean just remove them from the storybook description?

Reply:
sorry, i didn't see that we already used it before